Source/Scene/Model.js

```diff
@@ -1186,8 +1187,21 @@ define([
         var cacheKey = defaultValue(options.cacheKey, getAbsoluteUri(url));
 
         options = clone(options);
-        options.basePath = getBaseUri(url, true);
-        options.cacheKey = cacheKey;
+        var providedBasePath = options.basePath;
+        if (!defined(options.basePath)) {
```

Comment:
Set `defined(options.basePath)` to a variable so you don't call defined twice (lines 1191 and 1195)